lua/zenbones/shipwright/runners/alacritty.lua

```diff
@@ -1,11 +1,61 @@
+local helpers = require("shipwright.transform.helpers")
+local check_keys = {
+  "fg", "bg",
+  "cursor_fg", "cursor_bg",
+  "black", "red", "green", "yellow", "blue",
+  "magenta", "cyan", "white",
+  "bright_black", "bright_red", "bright_green", "bright_yellow", "bright_blue",
+  "bright_magenta", "bright_cyan", "bright_white",
+}
+
+local base_template = [[
+[colors.bright]
+black = "$bright_black"
+red = "$bright_red"
+green = "$bright_green"
+yellow = "$bright_yellow"
+blue = "$bright_blue"
+magenta = "$bright_magenta"
+cyan = "$bright_cyan"
+white = "$bright_white"
+
+[colors.cursor]
+cursor = "$cursor_bg"
+text = "$cursor_fg"
+
+[colors.normal]
+black = "$black"
+blue = "$blue"
+cyan = "$cyan"
+green = "$green"
+magenta = "$magenta"
+red = "$red"
+white = "$white"
+yellow = "$yellow"
+
+[colors.primary]
+background = "$bg"
+foreground = "$fg"]]
+
+local function alacritty_toml(colors)
+  for _, key in ipairs(check_keys) do
```

Comment:
This is here, because I based this on the [existing alacritty transform built into shipwright](https://github.com/rktjmp/shipwright.nvim/blob/master/lua/shipwright/transform/contrib/alacritty.lua).